akka-operator/templates/020-clusterrole.yaml

```diff
@@ -120,3 +120,14 @@ rules:
       - delete
       - patch
       - update
+  - apiGroups:
+      - "networking.k8s.io"
```

Comment:
Surprising, it definitely worked when we developed it.
I suggest that we merge this and cut a new 0.4.6 release. We are in a good spot for that because we haven't merged any new features since 0.4.5 yet, as far as I can see.

https://github.com/lightbend/akka-platform-operator/compare/v0.4.5...main

Reply:
I also tested it, but I may have been using it from SBR run rather than an installed operator

Reply:
doesn't matter, but I'm sure I'm tested it on EKS with installed operator